sormas-backend/src/main/java/de/symeda/sormas/backend/task/TaskService.java

```diff
@@ -114,7 +115,10 @@ public Predicate createUserFilter(CriteriaBuilder cb, CriteriaQuery cq, From<Tas
 
 		// National users can access all tasks in the system
 		User currentUser = getCurrentUser();
-		if (currentUser.hasAnyUserRole(UserRole.NATIONAL_USER, UserRole.NATIONAL_CLINICIAN, UserRole.NATIONAL_OBSERVER, UserRole.REST_USER)) {
+		final JurisdictionLevel jurisdictionLevel = currentUser.getJurisdictionLevel();
+		if (currentUser == null
+				|| (jurisdictionLevel == JurisdictionLevel.NATION && !currentUser.hasAnyUserRole(UserRole.POE_NATIONAL_USER))
```

Comment:
Same as in the conversation above, check for point of entry jurisdiction instead of specific user role

Reply:
i cannot check for that as only the POE_INFORMANT has pointOfEntry jurisdiction
what i can use instead is !UserRole.isPortHealthUser(currentUser.getUserRoles())

Reply:
Oh, you're right! Okay, maybe that's the way to go then. :)